sample_formatted_code.do_not_merge.c

```diff
@@ -0,0 +1,562 @@
+    // Cpp11BracedListStyle:false puts end brace onto separate line iff
+    // continuation line can hold all intializers. Bug? Feature?
+    struct pcap_stat current = {
+            .ps_recv = 12, .ps_drop = 2345, .ps_ifdrop = 9876, .ps_what = 134};
+    pcap_stat current = {
+            .ps_recv = 12, .ps_drop = 2345, .ps_ifdrop = 9876, .ps_what = 134};
+    // One designated initializer per line if it does not fit into one
+    // continuation line as BinPacking is disabled for designated intializer.
+    struct pcap_stat current = {.ps_recv = 12,
+            .ps_drop = 2345,
+            .ps_ifdrop = 9876,
+            .ps_what = 1345,
+            .ps_more = 333};
+    current = {.ps_recv = 12,
+            .ps_drop = 2345,
+            .ps_ifdrop = 9876,
+            .ps_what = 1345,
+            .ps_more = 333};
+
+    // function call continuation
+    function_call(
+            with, many, params, that, will spill, over, eventually, iff, max);
+    function_call(with, many, params, that, will spill, over, eventually, iff,
+            we, keep, on, adding);
+}
+
+struct Bitfields {
+    // clang 11: AlignConsecutiveBitFields
+    int aaaa : 1;
+    int b : 12;
+    int ccc : 8;
+};
+
+static void wrapping_literals()
+{
+    // string literal is too long. Continuation is from "string literal start",
+    // not ContinuationIndentWith, due to reasons?
+    SCLogInfo("running in 'auto' checksum mode. Detection of interface "
+              "state will require " xstr(CHECKSUM_SAMPLE_COUNT) " packets");
+
+    // Same as above but with additional parameter using ContinuationIndentWith
+    SCLogInfo("running in 'auto' checksum mode. Detection of interface "
+              "state will require " xstr(CHECKSUM_SAMPLE_COUNT) " packets %d",
+            someValue);
+
+    // Just params use ContinuationIndentWith
+    SCLogError(SC_ERR_STAT, "(%s) Failed to get pcap_stats: %s", tv->name,
+            pcap_geterr(ptv->pcap_handle));
+
+    // string literal param that fits on continuation line uses
+    // ContinuationIndentWith
+    SCLogError(SC_ERR_INITIALIZATION,
+            "Error getting context for Prelude. \"initdata\" argument NULL");
+    if (unlikely(initdata == NULL)) {
+        // string literal param does not fit on continuation line
+        // Continuation is from "string literal start", not
+        // ContinuationIndentWith, due to reasons
+        SCLogError(SC_ERR_INITIALIZATION, "Error getting context for Prelude.  "
+                                          "\"initdata\" argument NULL");
+        SCReturnInt(TM_ECODE_FAILED);
+    }
+
+    // function call inside if needs breaking apart and uses
+    // ContinuationIndentWith starting at function. Nice.
+    if (DetectContentDataParse("fileext", str, &fileext->ext, &fileext->len,
+                &fileext->flags) == -1) {
+        goto error;
+    }
+}
+
+//--- foreach handling as "for loop" ---
+void foreach_handling()
+{
+    // ForEachMacros
```

Comment:
Missing in code-style.rst.

Probably worth discussing at least in here as foreach() functions are currently handled inconsistently. See comments below in file.